CONTRIBUTING.md

```diff
@@ -0,0 +1,253 @@
+# Contributing to lisk-template
```

Comment:
This file needs to be customised for each individual project. At the very least `lisk-template` needs to be replaced with the name of your project, but ideally you would also add project-specific examples/extensions. Take a look at Commander's `CONTRIBUTING.md` for an example.